### ISSUE TITLE
Hover effect for table headers

### ISSUE THREAD

**Issue by rkeen-siemens** [JFormDesigner/FlatLaf#636]
It would be nice to have the ability to set hover colors for table row/column headers.

**Comment by DevCharly** [comment]
Implemented in latest `3.1-SNAPSHOT`: https://github.com/JFormDesigner/FlatLaf#snapshots

![grafik](https://user-images.githubusercontent.com/5604048/216814934-64444f7e-6628-4345-ad6b-0da6d9783c38.png)

There are 4 new (optional) UI properties:
~~~
TableHeader.hoverBackground
TableHeader.hoverForeground
TableHeader.pressedBackground
TableHeader.pressedForeground
~~~

**Comment by Kokecena** [comment]
Maybe it's a bit late, but it would also be interesting to have an hover effect when passing through the rows.

https://github.com/JFormDesigner/FlatLaf/assets/75457600/97844a3e-d62c-41fe-9c9c-a5134eea713d

Although I don't know how complicated it may be to implement or even if it is feasible to do so.
